pynamodb/expressions/condition.py

```diff
@@ -25,16 +25,30 @@ def serialize(self, placeholder_names, expression_attribute_values):
 
     def __and__(self, other):
         if not isinstance(other, Condition):
```

Comment:
sorry if this is a silly question, but why introduce `__rand__` and `__ror__`, instead of just allowing `other: Optional[Condition]`? Where it returns `self` if `other is None` and returns the joined condition if `isinstance(other, Condition)`

Reply:
I ask because when I first saw this question come up, it reminded me of all of the times I've had to write something along the lines of:
```
existing_condition = additional_condition & existing_condition if existing_condition is not None else additional_condition
```
:P

Reply:
This should allow you to write that as:
```
existing_condition &= additional_condition
```
The reason to introduce `__rand__` and `__ror__` is because without them `None & additional_condition` would raise a `TypeError` since `&` is not a valid operand on `None`.

Reply:
O wait, sorry, I think my misunderstanding was what `__rand__` and `__ror__` stood for :0 Didn't realize they were basically "reverse" statement checks, and assumed they had something to do with the `&=`/`|=` operators themselves--nevermind!